src/plugins.js

```diff
@@ -0,0 +1,117 @@
+'use strict'
+
+const { app, nativeImage } = require('electron')
+const { join } = require('path')
+const { warn, verbose, logger } = require('./common/log')
+const { uniq, pluck } = require('./common/util')
+const { promises: jsonld } = require('jsonld')
+
+
+class Plugins {
+  constructor(root, config) {
+    this.root = root || ARGS.home || app.getPath('userData')
```

Comment:
If we really want to (or need to) run this in both processes, it should go into common -- that way the tests will run in both processes; it's ok if some of the tests are unique to a given process type, but even the fact that file will be loaded in both processes during testing is beneficial.

Reply:
Yes, I better move it there! Another reason to have an instance in the main process (besides the context menu) is for the part that handles installing/updating/deleting via running the npm subprocess (I'm assuming this is not possible from the renderer)